samples/WebApi/README.md

```diff
@@ -0,0 +1,130 @@
+# Description
```

Comment:
I would make `# Building a WebAPI app with CoreRT` to be the very first line of the doc, and omit `# Description`.